lib/puppet/functions/vault_lookup/lookup.rb

```diff
@@ -187,10 +187,16 @@ def get_secret(client:, uri:, token:, namespace:, key:)
       raise Puppet::Error, append_api_errors(message, secret_response)
     end
     begin
-      if key.nil?
-        JSON.parse(secret_response.body)['data']
+      json_data = JSON.parse(secret_response.body)
+      puts "KEY=#{key} DATA=#{json_data}"
```

Comment:
@treydock Ah shoot... this `puts` should not be here.

Reply:
@natemccurdy My bad, I removed here: https://github.com/voxpupuli/puppet-vault_lookup/pull/78